alphashield/trading/backtester.py

```
@@ -17,72 +17,182 @@ class BacktestResult:
+                max_position=float(opt_cfg.get("max_position", 0.5)),
+                min_return=float(opt_cfg.get("min_return", 0.0)),
+            )
+        )
+
+        exec_cfg = self.config.get("execution", {})
+        spread_bps = exec_cfg.get("spread_bps", {})
+        commission = float(exec_cfg.get("commission_per_trade", 0.0))
+        adv_limit = float(exec_cfg.get("adv_limit", 0.10))
+
+        cov_cfg = self.config.get("coverage", {})
+        exp_ret = float(cov_cfg.get("exp_return_assumption", 0.10))
```

Ruff (0.14.1)

Local variable `exp_ret` is assigned to but never used

Remove assignment to unused variable `exp_ret`

(F841)